source/macos/AdaptiveCards/AdaptiveCards/Renderers/BaseCardElementRenderer.swift

```diff
@@ -36,26 +36,13 @@ class BaseCardElementRenderer {
         updatedView.identifier = NSUserInterfaceItemIdentifier(element.getId() ?? "")
         updatedView.isHidden = !element.getIsVisible()
         
-        // Input label handling
-        if config.supportsSchemeV1_3, let inputElement = element as? ACSBaseInputElement, let label = inputElement.getLabel(), !label.isEmpty {
-            let attributedString = NSMutableAttributedString(string: label)
-            if let colorHex = hostConfig.getForegroundColor(style, color: .default, isSubtle: false), let textColor = ColorUtils.color(from: colorHex) {
-                attributedString.addAttributes([.foregroundColor: textColor], range: NSRange(location: 0, length: attributedString.length))
+        if let inputElement = element as? ACSBaseInputElement {
+            updatedView.configureInputElements(element: inputElement, view: view)
+        } else {
+            updatedView.addArrangedSubview(view)
+            if view is ACRContentStackView {
+                view.widthAnchor.constraint(equalTo: updatedView.widthAnchor).isActive = true
             }
```

Comment:
Should this be in else?
Seems something to be common for all irrespective of ACSBaseInputElement

Reply:
For ACSBaseInputElement I add it in ACRContentStackView as for that we need to add error message below i, else we will have to again call another function from here to setup errorMessage